Userland/Libraries/LibCore/System.cpp

```diff
@@ -1183,7 +1186,7 @@ ErrorOr<void> socketpair(int domain, int type, int protocol, int sv[2])
 ErrorOr<Array<int, 2>> pipe2([[maybe_unused]] int flags)
 {
     Array<int, 2> fds;
-#if defined(__unix__)
+#if defined(__unix__) && !defined(AK_OS_WIN32)
```

Comment:
Hmm the else here is pretty sus. The flags are dropped on the floor. Maybe worth a fixmes to fcntl the flags in the else case?

Reply:
and a runtime warning please